api/envoy/type/http/v3/path_transformation.proto

```diff
@@ -20,7 +20,15 @@ option (udpa.annotations.file_status).package_version_status = ACTIVE;
 // This API is a work in progress.
 
 message PathTransformation {
+  // Action to take on a http request when the request path url has been modified by a transformation.
+  enum NormalizePathAction {
+    CONTINUE = 0;
```

Comment:
There should be a `DEFAULT` here to allow runtime control.

Reply:
Makes sense.